packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -877,8 +877,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             "/_summarizer",
             this,
             () => this.summaryConfiguration,
-            async (full: boolean, safe: boolean) => this.generateSummary(full, safe),
-            async (propHandle, ackHandle, refSeq) => this.refreshLatestSummaryAck(propHandle, ackHandle, refSeq),
+            // eslint-disable-next-line max-len
+            async (full: boolean, safe: boolean, summaryLogger: ITelemetryLogger) => this.generateSummary(full, safe, summaryLogger),
```

Comment:
this.generateSummary.bind(this) is better IMHO, same for another one

Reply:
Why do you prefer `bind` over arrow function?

Not sure if this has much weight, but I found [this style guide.](https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions)

If it's just too lengthy and redundant to type out all the individual args on both sides, could we just do something like?
```
async (...args) => this.refreshLatestSummaryAck(...args)
```

Reply:
Too verbose and need to change every time args are changed. ...args would work too

Reply:
I like ...args.  I'll clean these up on Monday.